tests/js/server/dump/dump-setup-common.inc

```diff
@@ -750,13 +752,13 @@ function createHybridDisjointSmartGraphSingleServer() {
 
   db[edgesSatA].save(eDocs);
   eDocs = [];
-  // from A to B
+  // from A to A &&  B to B
   for (let i = 0; i < size; ++i) {
-    eDocs.push({_from: docsANonSat.saved[i], _to: docsBNonSat.saved[i], value: String(i)});
-    eDocs.push({_from: docsANonSat.saved[i], _to: docsBNonSat.saved[(i + 1) % size], value: String(i)});
+    eDocs.push({_from: docsANonSat.saved[i], _to: docsANonSat.saved[i], value: String(i)});
```

Comment:
Is this intentional that we now only have self-referencing edges?
Before there was a Ring of vertices.

Reply:
It seems actually content does not matter.

Reply:
Content does not really matter. Just needed to remove them as they we're not allowed in our disjoint smart graph case.